lib/config.js

```diff
@@ -19,5 +19,18 @@ module.exports = function() {
     }
   };
 
-  return rc('couch_email_auth', defaultConfig);
+  var config = rc('couch_email_auth', defaultConfig);
+
+  if (typeof config.allowedRedirectHosts === 'string' &&
+      config.allowedRedirectHosts.trim() !== '') {
+    config.allowedRedirectHosts = config.allowedRedirectHosts.split(',').map(
+      function(host) {
+        return host.trim();
+      }
+    );
+  } else {
```

Comment:
```js

  config.allowedRedirectHosts = [];
  if (typeof config.allowedRedirectHosts === 'string' &&
      config.allowedRedirectHosts.trim() !== '') {
    config.allowedRedirectHosts = config.allowedRedirectHosts.split(',').map(
      function(host) {
        return host.trim();
      }
    );
  } 
  return config;
```

not sure if it is easier to read, probably quite subjective :)

as I said, feel free to merge